assets/src/scss/layout/navbar/_burger-menu.scss

```diff
@@ -11,16 +11,28 @@
   justify-content: flex-start;
   top: 0;
   left: 0;
-  inset-inline-start: -100%;
+  transition: inset-inline-start 0.2s ease-out, visibility;
```

Comment:
The transition to close through medium screens is so fast

Reply:
Yes still looking into it, I'll push a fix later.